pkg/gpu/ebpf/c/types.h

```diff
@@ -12,11 +12,14 @@ typedef enum {
     cuda_sync
 } cuda_event_type_t;
 
+#define MAX_CONTAINER_ID_LEN 129
```

Comment:
why 129? add a comment in code

Reply:
I've looked in other places of the code that use Cgroups and this was the common maximum length, although it wasn't documented. I tried to look for some specification on the maximum name but I couldn't find anything.